internal/factsengine/gatherers/hostsfile.go

```diff
@@ -0,0 +1,74 @@
+package gatherers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"sort"
+	"strings"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/trento-project/agent/internal/utils"
+)
+
+const (
+	HostsFactKey  = "hosts"
```

Comment:
I would call this `HostsGathererName` . The `FactKey` thing go obsolete, I would say.